packages/mermaid/src/diagrams/class/classDiagram.spec.ts

```diff
@@ -264,6 +264,86 @@ class C13["With Città foreign language"]
       const str = 'classDiagram\n' + 'note "test"\n';
       parser.parse(str);
     });
+
+    it('should handle note with "cssClass" in it', function () {
+      const str = 'classDiagram\n' + 'note "cssClass"\n';
```

Comment:
No need for extra concatenation, I totally understand that it is taken from the samples above, but latter are not the best for imitation.

Reply:
Very true, I now use template strings